contrib/NeonVM/stack.h

```diff
@@ -0,0 +1,22 @@
+#ifndef __STACK_H
+#define __STACK_H
+
+#include "cell.h"
+
+typedef struct tagTStack {
+    int top;
+    int capacity;
+    int max;
+    Cell **data;
+} TStack;
+
+struct tagTStack *createStack(int capacity);

```

Comment:
Again you've created a typedef for TStack but you're not using it here.

Reply:
Used typedef for all references.